sympy/physics/units/tests/test_quantities.py

```diff
@@ -60,15 +60,14 @@ def test_Quantity_definition():
     v =  Quantity("u")
     v.set_dimension(length)
     v.set_scale_factor(5*kilo)
-
     assert q.scale_factor == 10
     assert q.dimension == time
     assert q.abbrev == Symbol("sabbr")
-
     assert u.dimension == length
     assert u.scale_factor == 10
     assert u.abbrev == Symbol("dam")
-
+    if not isinstance(km.abbrev, str):
+        skip("Exception for Quantity(name, dimension, sacle_factor, abbrev) ")
```

Comment:
SymPy uses the following format to check if Deprecation Warning is raised: (See Line 78)
`raises(SymPyDeprecationWarning, lambda: Quantity('mismatch', dimension=length, scale_factor=kg))` 
I think it would be good to maintain uniformity.

Reply:
That block can actually be removed, I guess. There was an issue due to which I had to skip the test. Later I realized that it's just specific to my machine. There is no needed to skip that test. I'll update it.

Reply:
Yes, also, there was a typo in `scale_factor`.

Reply:
Yes, that'll be removed anyway. So no issues